src/App.php

```diff
@@ -22,30 +22,30 @@ public function __construct()
     public function run()
     {
         try {
-            $this->router->get();
+            $this->router->get($this);
         }catch(RouterException $e) {
             view('404', [
                 'error' => $e->getMessage()
             ]);
         }
     }
 
-    public function setErrorReporting()
+    private function setErrorReporting()
     {
         ini_set('error_reporting', E_ALL);
         
         ini_set('display_errors', $this->environment == 'development' ? 'On' : 'Off');
     }
 
-    public function loadConfigFile()
+    private function loadConfigFile()
```

Comment:
i have a strange error now ! 
i didn't change this function but now 
`
        $this->registry['config'] = require '../config/config.php'; 
`
this works fine on the browser but tests fails as it can't find this file
but when i make it 
`
 $this->registry['config'] = require 'config/config.php'; 
`
test passes but fails on the browser :)

Reply:
@vladimino I let the latest commit pass tests but it's not working on the browser  
Need to find a way for it to work on both + what's the point on my testing now right :laughing: ?

Reply:
Try to use `__DIR__` instead of relative paths.